library/core/src/cell.rs

```diff
@@ -82,6 +82,21 @@
 //!
 //! The corresponding [`Sync`] version of `OnceCell<T>` is [`OnceLock<T>`].
 //!
+//! ## `LazyCell<T, F>`
+//!
+//! A common pattern with OnceCell is, for a given OnceCell, to use a specific function on every
+//! call to [`OnceCell::get_or_init`] with that cell. This is what is offered by [`LazyCell`],
+//! which pairs cells of `T` with functions of `F`, and calls `F` before it yields `&T`.
```

Comment:
s/cells/a cell/ and s/functions/a function/ as far as I'm not missing anything?

Reply:
`T` and `F` are generic and thus there are many hypothetical cells and functions.

Reply:
Oh, I am stupid, didn't read it that way :)